documentation/modules/rn-2.3.adoc

```diff
@@ -0,0 +1,106 @@
+|The NIC interface type is not supported by OpenShift Virtualization (only e1000, rtl8139 and virtio interface types are currently supported).   |The migrated VM will have a virtio NIC if the source interface is not recognized.
+|The VM is using a vNIC profile configured for host device passthrough, which is not currently supported by OpenShift Virtualization.   |The migrated VM will have an SR-IOV NIC. The destination network must be set up correctly.
+|One or more of the VM's disks has an illegal or locked status condition.  |The migration will proceed but the disk transfer is likely to fail.
+|The VM has a disk with a storage type other than `image`, and this is not currently supported by OpenShift Virtualization.   |The migration will proceed but the disk transfer is likely to fail.
+|The VM has one or more snapshots with disks in ILLEGAL state. This is not currently supported by OpenShift Virtualization.   |The migration will proceed but the disk transfer is likely to fail.
+|The VM has USB support enabled, but USB devices are not currently supported by OpenShift Virtualization.  |The migrated VM will not have USB devices.
+|The VM is configured with a watchdog device, which is not currently supported by OpenShift Virtualization.   |The migrated VM will not have a watchdog device.
+|The VM's status is not `up` or `down`.  |The migration will proceed but it might hang if the VM cannot be powered off.
+|===
+
+.QEMU guest agent is not installed on migrated VMs
+
+The QEMU guest agent is not installed on migrated VMs. Workaround: Install the QEMU guest agent with a post-migration hook. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018062[*BZ#2018062*])
+
+.Importer pod log is unavailable after warm migration
+
+Retaining the `importer` pod for debug purposes causes warm migration to hang during the precopy stage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2016290[*BZ#2016290*])
+
+As a temporary workaround, the `importer` pod is removed at the end of the precopy stage so that the precopy succeeds. However, this means that the `importer` pod log is not retained after warm migration is complete. You can only view the `importer` pod log by using the `oc logs -f <cdi-importer_pod>` command during the precopy stage.
+
+This issue only affects the `importer` pod log and warm migration. Cold migration and the `virt-v2v` logs are not affected.
+
+.Deleting migration plan does not remove temporary resources.
+
+Deleting a migration plan does not remove temporary resources such as `importer` pods, `conversion` pods, config maps, secrets, failed VMs and data volumes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[*BZ#2018974*]) You must archive a migration plan before deleting it in order to clean up the temporary resources.
+
+.Unclear error status message for VM with no operating system
```

Comment:
@sbeskin-redhat , Here I see an example of an error message that is not clear - yet another example of such a case

Reply:
@istein1  So what should I do about that?